process-services/1.11/config/database.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Database configuration
+---
+
+Set the following properties to update the database properties.
+
+## JDBC connection
+
+Set the following properties to configure a JDBC connection:
+
+|Property|Description|
+|--------|-----------|
+|datasource.driver|The JDBC driver used to connect to the database. Note that the driver must be on the classpath of the web application.|
+|datasource.url|The JDBC URL used to connect to the database.|
+|datasource.username|The user of the database system that is used to connect to the database.|
+|datasource.password|The password of the above user.|
+
+Example:
+
+```text
+datasource.driver=com.mysql.jdbc.Driver
```

Comment:
for APS 2 please use `com.mysql.cj.jdbc.Driver`.

Reply:
Done for 2.0 cheers :-)